commons/Cargo.toml

```diff
@@ -19,11 +19,10 @@ serde.workspace = true
 serde_json.workspace = true
 serde_yaml.workspace = true
 once_cell = { workspace = true, optional = true }
-icu_locid = "=1.2.0"
-icu_segmenter = "=1.2.0"
-icu_provider = { version = "=1.2.0" }
-icu_provider_blob = "=1.2.0"
-icu_datagen = { version = "=1.2.5", default-features=false, features = [ "networking" ] }
+icu = "=1.2.0"
```

Comment:
I assume this is required by the module based icu data? It would be nice if we could only add dependencies that we use.

Reply:
yes. the module requires icu directly. i removed all dependencies that are included in the icu to keep it minimal